funcbench/main_test.go

```diff
@@ -15,6 +15,12 @@ package main
 
 import (
 	"testing"
+
+	fixtures "gopkg.in/src-d/go-git-fixtures.v3"
+	"gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/plumbing"
+	"gopkg.in/src-d/go-git.v4/plumbing/cache"
+	"gopkg.in/src-d/go-git.v4/storage/filesystem"
```

Comment:
can't we use the new location?
https://github.com/go-git/go-git

Reply:
Yes, I will come with more PRs on that later.
It is enough for current PR right now.